### PR TITLE
Count models with 0 speedup as failures

### PR DESCRIPTION
**Pull request by huydhn**
Reported by @xmfan where the dashboard counts models with 0 speedup (failed perf benchmark) as success.  They should be count as failures instead.

### Testing
https://torchci-git-fork-huydhn-failed-benchmark-models-fbopensource.vercel.app/benchmark/compilers

### CODE REVIEW

**Review by xmfan — APPROVED** [review]
LGTM, thanks for implementing the fix!

**Comment by huydhn** [comment]
@xmfan Pleae let me know when we're ready to merge this change.

**Comment by xmfan** [comment]
`pass_due_to_skip` might have an edge case:

we can skip a model's accuracy and perf tests and its accuracy will be marked as `pass_due_to_skip`, but perf results will show 0.0000 e.g. timm_vision_transformer_large

training, cudagraphs, torchbench passrate should be 62/65 instead of 61/65

**Comment by huydhn** [comment]
> `pass_due_to_skip` might have an edge case:
> 
> we can skip a model's accuracy and perf tests and its accuracy will be marked as `pass_due_to_skip`, but perf results will show 0.0000 e.g. timm_vision_transformer_large
> 
> training, cudagraphs, torchbench passrate should be 62/65 instead of 61/65

Oh I see.  That's an edge case indeed.  So `pass_due_to_skip` + 0 speed up would be treated as passing instead.

File: torchci/pages/benchmark/compilers.tsx [hunk 1]
```diff
@@ -210,7 +210,10 @@ function computePassrate(
       passCount[bucket][workflowId][suite][compiler] = 0;
     }
 
-    if (isPass(bucket, workflowId, suite, compiler, model, passingModels)) {
+    if (
+      isPass(bucket, workflowId, suite, compiler, model, passingModels) &&
+      record.speedup !== 0.0
```
Comment (ZainRizvi) [inline torchci/pages/benchmark/compilers.tsx:215]: Is there never a legitimate scenario where the speedup is zero?

**Review by ZainRizvi — APPROVED** [review]

File: torchci/pages/benchmark/compilers.tsx (hunk 1 above)
Comment (xmfan) [reply torchci/pages/benchmark/compilers.tsx:215]: It practically never happens, even when we run the same model twice. We could check for absolute execution time instead which may be a better indicator. I don't think it's worth adding an additional state to indicate whether or not the perf test errored out

Comment (ZainRizvi) [reply torchci/pages/benchmark/compilers.tsx:215]: Makes sense, though I'd suggest adding a comment explaining why we're ignoring the zero case and the shortcut being taken
